### PR TITLE
[issue-499] Complex type should support double 

### PR DESCRIPTION
**Pull request by Jay357089**

### CODE REVIEW

**Comment by Jay357089** [comment]
@Vimal-Das @QiangCai pls review and check in

**Comment by Jay357089** [comment]
retest this please

**Comment by QiangCai** [comment]
@Jay357089 
LGTM. please add test case.

**Comment by Jay357089** [comment]
@QiangCai handled. pls check

File: integration/spark/src/test/scala/org/carbondata/integration/spark/testsuite/complexType/TestCreateTableWithDoubleInComplex.scala [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+/**
+ * Test class for creating carbon table with double in complex type
+ *
+ */
+class TestCreateTableWithDoubleInComplex extends QueryTest with BeforeAndAfterAll {
+  override def beforeAll: Unit = {
+  }
+
+  test("test creating carbon table with double in complex type") {
```
Comment (nareshpr) [inline integration/spark/src/test/scala/org/carbondata/integration/spark/testsuite/complexType/TestCreateTableWithDoubleInComplex.scala]: Can u verify normal double column as dimension, 
i.e., by default double column will be considered as measure, can u dictionary_include double column as dimension and verify dataload and query flow ? also please add more testcase for double dimension.

**Comment by Jay357089** [comment]
@nareshpr handled. pls check.

**Comment by nareshpr** [comment]
LGTM

**Comment by ravipesala** [comment]
LGTM

**Pull request merged**